src/java/org/apache/cassandra/service/reads/trackwarnings/WarningContext.java

```diff
@@ -28,9 +28,9 @@
 
 public class WarningContext
 {
-    private static EnumSet<ParamType> SUPPORTED = EnumSet.of(ParamType.TOMBSTONE_WARNING, ParamType.TOMBSTONE_ABORT,
-                                                             ParamType.LOCAL_READ_SIZE_WARN, ParamType.LOCAL_READ_SIZE_ABORT,
-                                                             ParamType.ROW_INDEX_SIZE_WARN, ParamType.ROW_INDEX_SIZE_ABORT);
+    private static EnumSet<ParamType> SUPPORTED = EnumSet.of(ParamType.TOMBSTONE_WARNING, ParamType.TOMBSTONE_FAIL,
```

Comment:
Should we rename the `org.apache.cassandra.service.reads.trackwarnings` package to `org.apache.cassandra.service.reads.readthresholds`, or perhaps `org.apache.cassandra.service.reads.thresholds`?

Reply:
how about `package org.apache.cassandra.service.thresholds;`?  CASSANDRA-17258 adds a write threshold (blocked on 2 JIRAs), so rather than a `read.thresholds` and `write.thresholds` we just have `thresholds`?

Reply:
`org.apache.cassandra.service.thresholds` makes sense to me